lablgtk3-sourceview3.opam

```diff
@@ -14,7 +14,7 @@ license: "LGPL with linking exception"
 depends: [
   "ocaml"                {         >= "4.05.0"  }
   "dune"                 { build & >= "1.4.0"   }
-  "lablgtk3"             {            "0.beta3" }
+  "lablgtk3"             {         >= "0.beta4" }
```

Comment:
Actually, this is going to be 3.0.beta4, due to the handling of tags (since lablgtk2 and 3 share the same repository, tags such as 0.beta4 seem a bad idea).

Reply:
Fixed, also amended the README to take into account #44